.env.example

```diff
@@ -19,6 +19,7 @@ ALLOW_PAGOPA_IP_SOURCE_RANGE="::ffff:ac13:1/112"
 AUTHENTICATION_BASE_PATH=""
 API_BASE_PATH="/api/v1"
 PAGOPA_BASE_PATH="/pagopa/api/v1"
-PAGOPA_API_URL= ""
+PAGOPA_API_URL=""
+PAGOPA_API_URL_TEST=""
```

Comment:
this new variable need to be documented in the README

Reply:
also I just noticed that [we don't set these two variables during tests](https://github.com/teamdigitale/io-backend/blob/master/.circleci/config.yml#L52), we should

Reply:
ok, i need the values to add in .env.example and/or in `config.yml`
if values need to be hidden we can set those like project env variables inside [circleci](https://circleci.com/gh/organizations/teamdigitale/settings#projects) i don't have privileges

Reply:
Variables information added inside README with commit 204313b

Reply:
you can just use dummy values since they are just used for tests, like `https://pagopa-proxy` and `https://pagopa-proxy-test`